test-app/apps/backend-v2/src/app/utils/date.ts

```diff
@@ -0,0 +1,64 @@
+import { z, ZodString, ZodError } from "nestjs-zod/z";
+
```

Comment:
A lot of this also looks copied. Can it just be imported from the shared package instead?

Reply:
Same comment